index.html

```diff
@@ -241,5 +241,30 @@ <h4 class="connection2">LET'S CONNECT</h4>
         </div>
     </section>
 
+    <section class="contact-form">
+        <div class="contact-container">
+            <h2 class="contact-me">Contact me</h2>
+            <p class="contact-text">
+                If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding. 
+                I’d love to help with it!
+            </p>
+        </div>
+
+        <form action="https://formspree.io/f/meqwrnzl" method="post">
+            <ul class="contact-list">
+                <li>
+                    <input type="text" id="name" placeholder="Full name" maxlength="30" required />
+                </li>
+                <li>
+                    <input type="email" id="email" placeholder="Email Adress" required />
+                </li>
+                <li>
+                    <textarea rows="6" cols="40" id="text" placeholder="Write your message here..." maxlength="500"></textarea>"
+                </li>
+                <button type="submit" id="contact-submit">Get in touch</button>
+            </ul>
```

Comment:
- [ ] While trying to submit the form with some information, I notice that the form spree throws an error, I would kindly suggest you add the name attribute to all the input form. On the `textarea`, kindly put the `required` attribute

here is the error 👇 

![form](https://user-images.githubusercontent.com/78634841/235684600-f86ffe9e-f9a6-4dae-93f5-6b0b333d86c9.png)